packaging/deb-in/deb_debug.cmake

```diff
@@ -117,14 +117,6 @@ usr/lib/mysql/plugin/debug/component_test_udf_registration.so
 usr/lib/mysql/plugin/debug/component_test_host_application_signal.so
 usr/lib/mysql/plugin/debug/component_test_mysql_current_thread_reader.so
 usr/lib/mysql/plugin/debug/component_test_mysql_runtime_error.so
-<<<<<<< HEAD
-||||||| merged common ancestors
-usr/lib/mysql/plugin/debug/component_test_component_deinit.so
-=======
-usr/lib/mysql/plugin/debug/component_test_component_deinit.so
-usr/lib/mysql/plugin/debug/component_test_mysql_command_services.so
-usr/lib/mysql/plugin/debug/test_services_command_services.so
```

Comment:
Any reasons why `component_test_component_deinit.so` was omitted in the first place? Can we have PXC inherit new changes as well?

Reply:
Done.